### DISCUSSION TITLE
x-listbox should have the possibility to be tabbed when the options panel is open

### DISCUSSION THREAD

**Discussion by joffreypersia** [alpinejs/alpine#3885] [3. Feature ideas]
Hi, 

I don't get why when we use `x-listbox`, the `tabindex="-1"` and there is no focus trap inside. 

The documentation says that it only allows arrow key to navigate, but usually, the lambda user knows tab.

Can the `tabindex` be made possible and have a focus trap without rewriting the entire headless component? 

Documentation : https://alpinejs.dev/component/headless-listbox/docs

Thank you

**Comment by ekwoka** [comment]
I believe the behavior of the listbox component is specced to be the same as a native select field.

A native select field doesn't allow tabbing, you have to use the arrows.

**Comment by hirasso** [comment]
@ekwoka thank you for your very active participation in Alpine's GitHub issues & discussions.

What do you mean with "I beliefe..."? Are you one of Alpine's core team members? If not - could we get an _official_ answer from someone that makes money with the headless components ( @calebporzio ?...)

Right now, the support experience for Alpine's premium components is not very transparent IMHO, which makes it hard to use them in production with confidence.
